src/test/java/com/napier/sem/MyTest.java

```diff
@@ -0,0 +1,60 @@
+package com.napier.sem;
+
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestInstance;
+
+import java.sql.Connection;
+import java.util.ArrayList;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+public class MyTest {
+    static App app;
+
+    // Connection to MySQL database.
+    final static Connection connection = null;
+
+    // Create instance of the database access object
+    // For some reason this doesn't work, it has to be called from the App
+    // DAO dao = new DAO(MyTest.connection);
+    DAO dao = new DAO(App.connection);
+
+
+    @BeforeAll
+    static void init() {
+        // Create new Application
+        App app = new App();
+        // Connect to database
+        app.connect("localhost:33061");
+    }
+
+    @Test
+    void topNCitiesInTest1()
```

Comment:
it'd be better practice to name the test after what it's actually testing. for this one it could be `topNCitiesInArrayIsCorrectSize` or something similar.